agent/exec/tail_buffer_test.go

```
@@ -0,0 +1,246 @@
+		{"first over limit, second small", 200, 10},
+		{"both tiny", 5, 5},
+	}
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			tb := newTailBuffer(limit)
+			all := make([]byte, 0, tc.firstWrite+tc.secondWrite)
+			a := bytes.Repeat([]byte("A"), tc.firstWrite)
+			b := bytes.Repeat([]byte("B"), tc.secondWrite)
+			all = append(all, a...)
+			all = append(all, b...)
+			tb.Write(a)
```

golangci-lint (2.12.2)

[error] Error return value of `tb.Write` is not checked

(errcheck)